extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -282,26 +272,23 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
 
   /**
    * Constructs block-level url parameters with side effect of setting
-   * size_, jsonTargeting_, and adKey_ fields.
+   * size_ and adKey_ fields.
    * @return {!Object<string,string|boolean|number>}
    */
   getBlockParameters_() {
     dev().assert(this.initialSize_);
-    dev().assert(this.jsonTargeting_);
     let sizeStr = `${this.initialSize_.width}x${this.initialSize_.height}`;
-    const tfcd = this.jsonTargeting_ && this.jsonTargeting_[TFCD];
-    const multiSizeDataStr = this.element.getAttribute('data-multi-size');
+    const multiSizeDataStr = this.getStringData('multiSize');
     if (multiSizeDataStr) {
-      const multiSizeValidation = this.element
-          .getAttribute('data-multi-size-validation') || 'true';
+      const multiSizeValidation = this.getStringData('multiSizeValidation');
       // The following call will check all specified multi-size dimensions,
       // verify that they meet all requirements, and then return all the valid
       // dimensions in an array.
       const dimensions = getMultiSizeDimensions(
           multiSizeDataStr,
           this.initialSize_.width,
           this.initialSize_.height,
-          multiSizeValidation == 'true',
+          !multiSizeValidation || multiSizeValidation == 'true',
```

Comment:
Why do you need this?  multiSizeValidation equal to '', undefined, or null would not == 'true'